### PR TITLE
Fix saving and loading document

### PR DESCRIPTION
**Pull request by cdrake**
List of fixed issues (if they exist):
- Drag and dropping document was not loading config options
- Loading tractography, and connectomes 
- Saving color map labels

### CODE REVIEW

**Review by hanayik — CHANGES_REQUESTED** [review]
@cdrake , there are a few `console.log` lines that look like they were used for debugging. Can you convert these to use the niivue logger instead if they aren't critical logs?

File: src/index.html
Comment (neurolabusc) [inline src/index.html:1]: @cdrake I only changed this web page to help test your new features. I would not change the current NiiVue ./src/index.html file as it is nice to have a minimal page when using `npm run dev`. Beyond this, this is terrific work.

Comment (cdrake) [reply src/index.html:1]: Yes, index.html should be reverted.

**Comment by cdrake** [comment]
> @cdrake , there are a few `console.log` lines that look like they were used for debugging. Can you convert these to use the niivue logger instead if they aren't critical logs?

Console.logs added should be removed or converted to log.error

**Pull request merged**